test/unit/writer/settings.test.ts

```diff
@@ -72,6 +80,27 @@ describe('writer/settings', () => {
       });
       it.todo('rejects if the submitter does not have permission');
       it.todo('rejects if the submitter does not have permission to change admin');
+
+      it('rejects if passing more than 8 strategies for normal space', async () => {
+        return expect(
+          verify(
+            editedInput({
+              strategies: randomStrategies(10)
+            })
+          )
+        ).rejects.toContain('wrong space format');
+      });
+
+      it('rejects if passing more than 10 strategies for turbo space', async () => {
+        mockGetSpace.mockResolvedValueOnce({ ...spacesGetSpaceFixtures, turbo: true });
+        return expect(
+          verify(
+            editedInput({
+              strategies: randomStrategies(12)
+            })
+          )
+        ).rejects.toContain('wrong space format');
+      });
     });
```

Comment:
Every time we want to increase number of strategies limit in snapshot.js, we have to change the limits here 🙈can i change it to pick from schema?

Reply:
Yes, was not aware it was exported

Reply:
or should be fine maybe?

Reply:
Dynamic will be better